covert/__main__.py

```diff
@@ -114,7 +114,10 @@ def argparse():
     sys.exit(1)
 
   aiter = iter(av[1:])
+  longargs = [flag[1:] for switches in ad.values() for flag in switches if flag.startswith("--")]
```

Comment:
Should be `flag[2:]` if you wish to remove both hyphens (I did not yet look where you might be using this).